config/Makefile.msvc

```diff
@@ -97,12 +97,11 @@ AFL_INSTRUMENT=false
 ########## Configuration for the bytecode compiler
 
 ### Which C compiler to use for the bytecode interpreter.
-CC=cl
-BYTECFLAGS=-nologo -D_CRT_SECURE_NO_DEPRECATE -O2 -Gy- -MD
-BYTECODE_C_COMPILER=$(CC) $(BYTECFLAGS)
-
-BYTECCCOMPOPTS=-DCAML_NAME_SPACE
-
+CC = cl
+CFLAGS = -nologo -O2 -Gy- -MD
+CPPFLAGS = -D_CRT_SECURE_NO_DEPRECATE -DCAML_NAME_SPACE
+OCAMLC_CFLAGS = -nologo -O2 -Gy- -MD
+OCAMLC_CPPFLAGS = -D_CRT_SECURE_NO_DEPRECATE
```

Comment:
It might make sense to put this in the mingw* config files too. Mingw-w64 has a couple references to _CRT_SECURE_NO_DEPRECATE which don't do anything but it might in the future. Since this option is not related to a given compiler but to a platform, it's better to have it in all win32-related places. Moreover there are mix and matches (in particular, some clang setups might MSVC headers).